lib/ansible/plugins/doc_fragments/action_core.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) , Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+# WARNING: this is mostly here as a convinence for documenting core behaviours, no plugin outside of ansbile-core should use this file
+class ModuleDocFragment(object):
+
+    # requires action_common
+    DOCUMENTATION = r'''
+attributes:
+    async:
+      support: none
+    become:
+      support: none
+    bypass_task_loop:
+      description: these tasks ignore the ``loop`` and ``with_`` keywords
+      support: none
+    core:
+      description: This is a 'core engine' feature and is not implemented like most task actions, so it is not overridable in any way via the plugin system.
+      support: full
+    connection:
+      support: none
+    ignore_conditional:
```

Comment:
Either remove this, or add some values, or at least add `{}` to make its value a dictionary (and not `None`).
```suggestion
```